Assets/Scripts/Server/AvatarBehaviour.cs

```diff
@@ -24,6 +24,11 @@ public void SetPlayerDrivenMovement(List<Direction> directions)
     }
   }
 
+  public bool IsAlive()

```

Comment:
This is just stylistic. But in C# a method like IsAlive() is going to read like an assigner. The getter version would be either GetIsAlive() or, more likely, a property getter:
> public bool IsAlive { get { return !_dead; } }